### PR TITLE
Ft crud company

### PR DESCRIPTION
**Pull request by NyiingangoPacifique**
#### What does this PR do?

- Create,Read,Update and Delete Company

#### Description of Task to be completed?

- An admin or operator must be able to do all CRUD function for Company and view error or success at any attempt and system should have a skelton.

#### How should this be manually tested?
**First clone this project and**

- checkout on branch

`git checkout origin ft-crud-company`

- next we need to make few change on `.env` if you don't have if you need to create in on project root

1. `REACT_APP_BACKEND_URL=https://phantom-api-pirate.herokuapp.com/
this is backend url
or use  backend port to run it locally which is the only way working now

3. PORT=your_port`

### CODE REVIEW

**Review by christianR15c — APPROVED** [review]

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -1,64 +1,38 @@
...
-                    {t("navbar.signIn")}
-                  </button>
-                </>
-              )}
+              <button className="w-[90px] border-button-color border-fon text-white bg-transparent hover:bg-button-color hover:text-white hover:border-none">
+                <Link to='/login'>
+                <a>{t('navbar.signIn')}</a>
+                </Link>
+              </button>
               <div className="px-10">
```
Comment (Anguandia) [inline src/components/Navbar.jsx]: We still need the conditional rendering of the logout/signin button to remind the user if they try to access the site when they are still logged in. Please restore that

File: src/components/check.jsx [hunk 2]
```diff
@@ -0,0 +1,296 @@
+import React, { useState, useEffect, useRef } from 'react';
```
Comment (Anguandia) [inline src/components/check.jsx]: This entire file seems redundant, but we'll deal with it later

**Review by Anguandia — CHANGES_REQUESTED** [review]
Good work, so impressive!!! Please pay attention to the few requested changes, also mind squashing your commits where possible.

**Review by christianR15c — APPROVED** [review]

File: src/components/check.jsx (hunk 2 above)
Comment (NyiingangoPacifique) [reply src/components/check.jsx]: resolved i removed it i wasn't used

File: src/components/Navbar.jsx (hunk 1 above)
Comment (NyiingangoPacifique) [reply src/components/Navbar.jsx]: logout added

**Review by Anguandia — CHANGES_REQUESTED** [review]
Almost there, one last thing though:
dropDownMenu is undefined(line 67 & 76), the page can not display due to errors in compilation. Please ensure you declare/define it, everything else will be just fine

**Review by Anguandia — APPROVED** [review]

**Review by SimonImanigirimpuhwe — APPROVED** [review]
LGTM

**Pull request merged**